dev-guide.md

```diff
@@ -27,6 +26,17 @@ To check out the code, you first need to [install Git](https://git-scm.com/book/
 git clone git@github.com:cardinalitypuzzles/smallboard.git
 ```
 
+### Set up an `.env` file
+
+Both Docker and manual setup read environment variables from an `.env` file. An example `.env.test` file is provided, which you should copy to `.env`:
+
+```
+cp .env.test .env
+```
+
+See below for more details on the `.env` file.
```

Comment:
Should we put more details on how to get the necessary API keys? (Google, Discord)

Like -- "Contact these people if you want API keys" or something

Reply:
Good idea, I'll add something about that. (I was planning to just grab all the keys from our Heroku deployment :)

Reply:
Hmm, in later sections, it already says to contact a collaborator for env settings like keys or to just get them from https://dashboard.heroku.com/apps/smallboard/settings

I'd rather not put a specific person since that might fall out-of-date over time